{{cookiecutter.project_name}}/server/apps/main/templates/main/index.html

```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="{% static 'main/css/index.css' %}">
+    <link rel="stylesheet" href="{% static 'main_app/css/index.css' %}">
```

Comment:
Good catch! But, I prefer consistent naming. So, let's better rename `static/main_app` to `static/main`

Reply:
Guess, @DmytroLitvinov is already on it. Thanks!

Reply:
Thank you for your attention :)